<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>About</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-1.1/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-1.1/highlight.js"></script>
<link href="site_libs/font-awesome-4.5.0/css/font-awesome.min.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Home</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="mailto:&lt;austin29hei@gmail.com&gt;">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://github.com/AustinYue/">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">About</h1>

</div>


<div id="education" class="section level3">
<h3>EDUCATION:</h3>
<hr />
<div id="xiamen-university-xiamenchina-082013-062017" class="section level5">
<h5><em>Xiamen University Xiamen,China</em> | 08/2013-06/2017</h5>
<p>● Major: Biological Technology<br />
● Degree: Bachelor of Science<br />
● GPA Ranking: 7/160</p>
</div>
<div id="national-sun-yat-sen-university-taiwan-092015-012016" class="section level5">
<h5><em>National Sun Yat-sen University Taiwan</em> | 09/2015-01/2016</h5>
<p>● Exchange Student majored in Biology</p>
</div>
<div id="johns-hopkins-university-072016-082016" class="section level5">
<h5><em>Johns Hopkins University</em> | 07/2016-08/2016</h5>
<p>● Short-term research training</p>
</div>
</div>
<div id="research-experience" class="section level3">
<h3>RESEARCH EXPERIENCE:</h3>
<hr />
<div id="short-term-research-training-joel-schildbachs-lab-johns-hopkins-university-072016-082016" class="section level5">
<h5><em>Short-Term Research Training, Joel Schildbach’s Lab, Johns Hopkins University</em> | 07/2016-08/2016</h5>
<p>● First Phase: Phage Research<br />
● Phage Enrichment: successfully collected soil sample and enrich the phages in moist soil<br />
● Phage Plaque Streak and Plates Dilution: Successfully obtained the purified phage through normal streak, T Streak and dilutions<br />
● Acquired MTL and HTL successfully: Increased the phage concentration from medium to high and calculated the titer<br />
● NanoDrop and Run QC Gel: Purified phage genome and did NanoDrop, Ran QC Gel<br />
● Second Phase: Independent-design experiment:Simple Experiment about The Effects of Different Calcium Concentration on The Growing of Phages<br />
● Prepared Calcium solutions with different concentration and labeled them. Prepared and labeled six bacteria tubes.<br />
● Added each different concentration Calcium solution 90μL to each corresponding bacteria tube<br />
● Added each Calcium-Bacteria mixture 10μL to HTL and Stored them at 4℃ overnight<br />
● Added top agar(TA) 4.5ml to each tube.Plated out and swirled the plates.<br />
● Results: Too much Calcium may have negative influences on the growing of phages.</p>
</div>
<div id="the-11th.-statistics-design-competition-in-xiamen-university-first-prize-052016" class="section level5">
<h5><em>The 11th. Statistics Design Competition in Xiamen University First Prize</em> | 05/2016</h5>
<p>Project: The Relation between high incidence rate of Lung Cancer and Air Pollution<br />
● Role: Group Leader<br />
● Collected annual data report of pollutant and incurrence of lung cancer in Beijing and Xiamen from 2007-2014 and annual data of forestation rate in Beijing and Xiamen<br />
● Conducted dimension-reduced processing of data<br />
● Used SPSS Regression Models to analyse the statistics<br />
● Established Panel Data Model to work out the relation between incidence rate of Lung Cancer and Air Pollution</p>
</div>
<div id="long-term-research-training-jizhiliangs-bioinformatics-lab-xiamen-university-092015-092016" class="section level5">
<h5><em>Long-Term Research Training, JiZhiliang’s Bioinformatics Lab, Xiamen University</em> | 09/2015-09/2016</h5>
<p>● First Phase:MOE Modeling of Frizzled-8 Protein<br />
● Utilisedprosite to anticipate protein domain and annotated every domain.<br />
● Analyzed every domain and confirmed their 3D plot<br />
● Used KEGG to find information of correlated protein<br />
● MOE Modellingand analysed the result of modelling.<br />
● Second Phase:Combined Project of ORF Finder, BLAST, PIR, SMART and Pfam<br />
● Through ORF Finder find the encode of Protein Sequences<br />
● Utilized BLAST to test accuracy of the amino acid sequence<br />
● Utilized PIR and correlated protein name to search corresponding proteins and wrote the entry names in Uniref<br />
● Utilized SMART or Pfam and entry names to search and wrote family name or Pfam ID of corresponding protein</p>
</div>
<div id="national-college-students-innovative-competition-the-preliminary-research-about-the-inhibition-effect-of-lipoxins-to-endometriosisemt-012015-012016" class="section level5">
<h5><em>National College Student’s Innovative Competition: The Preliminary Research about the Inhibition Effect of Lipoxins to Endometriosis(EMT)</em> | 01/2015-01/2016</h5>
<p>● Studied the expression of transcriptions factors-ZEB1,Slug-in endometriosis patients<br />
● Studied the expression of LXA4 in endometriosis patients<br />
● Based on selected clinical samples, at vitro molecular level, this project attempted to figure out whether EMT exist among endometriosis patients and analyzed the inhabitation effect of Lipoxins</p>
</div>
<div id="research-intern-at-chinese-academy-of-sciences-chengdu-branch-072015-082015" class="section level5">
<h5><em>Research Intern at Chinese Academy of Sciences, Chengdu Branch</em> | 07/2015-08/2015</h5>
<p>● Role: Lab technician<br />
● Practiced methods of culturing and screening cells; ELISA，SDS-PAGE;sequence Alignment<br />
● Interned in Chengdu Di Ao Medicine Group Co. Ltd. for a short time</p>
</div>
</div>
<div id="publications" class="section level3">
<h3>PUBLICATIONS:</h3>
<hr />
<p>Wu,Rongfeng; Yang,Huiming; Zhou,Weidong;Yue,Leiyu;Zhang,Lirong etc.2016.“Proteomic analysis of the effect of IL-1β and Lipoxin A4 in human endometriotic stromal cells” The Journal of Obstetrics and Gynaecology Research 2016(4)</p>
</div>
<div id="honors" class="section level3">
<h3>HONORS:</h3>
<hr />
<p>Annual Merit-based Scholarship in 2015-2016 Academic year | 06/2016<br />
Annual Merit-based Scholarship in 2013-2014 Academic year | 06/2014<br />
Scholarship for Social Practice in 2013-2014 Academic year | 06/2014</p>
</div>
<div id="skills" class="section level3">
<h3>SKILLS:</h3>
<hr />
<p>Computer Skills: R Language, C Language, SPSS,SAS, JAVA</p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
